pkg/cmd/step/step_unstash.go

```diff
@@ -121,14 +123,16 @@ func (o *StepUnstashOptions) Run() error {
 // CreateBucketHTTPFn creates a function to transform a git URL to add the token for accessing a git based bucket
 func CreateBucketHTTPFn(authSvc auth.ConfigService) func(string) (string, error) {
 	return func(urlText string) (string, error) {
-		token, err := GetTokenForGitURL(authSvc, urlText)
-		if err != nil {
-			log.Logger().Warnf("Could not find the git token to access urlText %s due to: %s", urlText, err)
-		} else if token != "" {
-			idx := strings.Index(urlText, "://")
-			if idx > 0 {
-				idx += 3
-				urlText = urlText[0:idx] + token + "@" + urlText[idx:]
+		if authSvc != nil {
```

Comment:
so not passing an auth service is not an error condition?

Reply:
not sure - i probably didn't need to change this.